.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Feature Request
+about: Make suggestions on how TM:PE could improve.
+title: ''
+labels: 'feature'
+assignees: ''
+---
+
+<!-- Before submitting this feature, please search and make sure it hasn't already been suggested, thanks! -->
+
+### Description
+<!-- In a few sentences, summarize what your idea is. -->
+
+
+### Sources / Links
+<!-- Post any links below that describe how your issue works. For example, with turn on red it'd be the link to a "turn on red rule" in a driver's handbook; things like that. -->
+
+
+### Screenshots
+<!-- If you have screenshots showing how this feature works, put them below. How to: https://bit.ly/2Kc8owO -->
+
+
+### Additional Info
+<!-- Any other things worthy of being noted? Let us know what they are below. -->
+
+
+<!--
+EXAMPLE:
```

Comment:
Again, not sure an example is helpful; it just adds visual clutter to the 'new issue' form.